code/modules/mob/observer/freelook/marker/signal_abilities/runes.dm

```diff
@@ -12,7 +12,7 @@
 /datum/signal_ability/runes/on_cast(var/mob/user, var/atom/target, var/list/data)
 	GLOB.cult.powerless = TRUE //Just in case. This makes sure the runes don't do anything
 	new /obj/random/rune(target)
-
+	link_necromorphs_to(SPAN_NOTICE("[user] placed a rune at LINK"), target)
```

Comment:
this would be extremely spammy, doesn't need a notification, people often mass place runes in large batches

Reply:
That's actually the entire point. It's LRP if done in mass spammings, and gives people probable cause to early metagame. This serves as an accurate way for players to ask admins for help.

Reply:
do we have rules about rune usage?

this is still an extremely spammy and un-optimal way to do it. if you want signals to know who made a rune, have that information stored and shown on examine

Reply:
I'll do that instead. Next week, anyway.